x-pack/plugins/security_solution/server/endpoint/ingest_integration.ts

```diff
@@ -57,9 +70,18 @@ export const getPackageConfigCreateCallback = (
     try {
       return updatedPackageConfig;
     } finally {
-      // TODO: confirm creation of package config
-      // then commit.
-      await manifestManager.commit(wrappedManifest);
+      if (snapshot.diffs.length > 0) {
```

Comment:
nit -> I think it will be nice to have try/catch here to avoid bubble up error and have a better way to handle it